pkg/supportbundle/staticspecs/embed.go

```diff
@@ -0,0 +1,58 @@
+package staticspecs
+
+import (
+	_ "embed"
+
+	troubleshootv1beta2 "github.com/replicatedhq/troubleshoot/pkg/apis/troubleshoot/v1beta2"
+	"github.com/replicatedhq/troubleshoot/pkg/supportbundle"
+)
+
```

Comment:
Please add the following comment here. I reckon someone (like me) would one day try to optimise this code by caching the parsed spec.

```go
// DO NOT CACHE THE PARSED SPEC
// All getters always parse the bundle from the raw spec (embedded files)
// to ensure that we fetch the latest specs defined in URI fields (if any)
```

Reply:
Add